plugins/samplemimo/metismiso/metismisoudphandler.cpp

```diff
@@ -622,8 +622,6 @@ void MetisMISOUDPHandler::processIQBuffer(unsigned char* buffer)
                 }
             }
 
-            sampleMic  = (int)((signed char) buffer[b++]) << 8;
-            sampleMic += (int)((unsigned char)buffer[b++]);
```

Comment:
This is a breaking change and I don't see the point of it.

Reply:
The variable "sampleMic" isn't used anywhere, so compiler complained about it, and thus I've removed it.

Reply:
b is modified and used though.

Reply:
Yes, you're right! Didn't noticed that.